dali/pipeline/data/buffer.h

```diff
@@ -86,71 +86,6 @@ class DLL_PUBLIC Buffer {
   inline Buffer() = default;
   virtual ~Buffer() = default;
 
-  /**
```

Comment:
This was just moved from `public:` to `protected:`

Reply:
I don't like it. The `Buffer` class is really quite useless after this change. How about making the inheritance of TensorList from Buffer protected (or even private) instead?

Reply:
useless -> redundant ?

Reply:
Switched to private inheritance.